lib/kafka_ex.ex

```diff
@@ -46,6 +46,53 @@ defmodule KafkaEx do
   end
 
   @doc """
+  Returns a KafkaEx worker child specification.
+
+  This child specification can be used to host KafkaEx workers outside of the
+  KafkaEx application itself, for example in your own application supervision
+  tree. Use together with the application option `disable_default_worker` set
+  to `true` to make sure no processes are started by the KafkaEx
+  application.
+
+  It takes the same arguments as create_worker/2.
+
+  Returns {:error, error_description} on invalid arguments
+
+  ## Example
+
+  ```elixir
+  iex> KafkaEx.worker_spec(:pr) # where :pr is the name of the worker created
+  {:ok,
```

Comment:
Maybe by way of example, just show this getting used in a supervisor?  All of the structure here doesn't really help now that this isn't a doctest.  Something like

```
{:ok, spec} = KafkaEx.worker_spec(:pr)
supervisor_opts = [strategy: :one_for_one, name: MySupervisor]
Supervisor.start_link(spec, supervisor_opts)
```